include/ck_tile/ops/gemm/pipeline/gemm_pipeline_ag_bg_cr_comp_v3.hpp

```diff
@@ -52,7 +52,7 @@ struct GemmPipelineAgBgCrCompV3 : public BaseGemmPipelineAgBgCrCompV3<Problem>
     using BLayout = remove_cvref_t<typename Problem::BLayout>;
     using CLayout = remove_cvref_t<typename Problem::CLayout>;
 
-    using BlockGemm = remove_cvref_t<decltype(Policy::template GetBlockGemm<Problem>())>;
+    using BlockGemm = typename decltype(Policy::template GetBlockGemmType<Problem>())::type;
```

Comment:
should use dedicated type alias like `typename Policy::BlockGemm` rather than expose the imeplementation details here (the helper function `GetBlockGemmType()` and `type_identity<>` object creation)